src/OpenTelemetry.Contrib.Exporter.Stackdriver/OpenTelemetry.Contrib.Exporter.Stackdriver.csproj

```diff
@@ -3,12 +3,11 @@
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <Description>Stackdriver .NET Exporter for OpenTelemetry.</Description>
     <PackageTags>$(PackageTags);Stackdriver;Google;GCP;distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Exporter.Stackdriver-</MinVerTagPrefix>
```

Comment:
could you validate locally that "." in the prefix is okay.

Reply:
Yup. `.` is okay